<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1816</width>
    <height>1231</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>宋体</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Plot 2D/3D</string>
  </property>
  <property name="windowIcon">
   <iconset resource="QtApp/res.qrc">
    <normaloff>:/icons/images/panda.png</normaloff>:/icons/images/panda.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_6">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>220</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>560</width>
           <height>0</height>
          </size>
         </property>
         <property name="title">
          <string>Line interval</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="topMargin">
           <number>5</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Interval</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QPushButton" name="btnRefreshData">
            <property name="text">
             <string>Replot</string>
            </property>
            <property name="icon">
             <iconset resource="QtApp/res.qrc">
              <normaloff>:/icons/images/rotateleft.png</normaloff>:/icons/images/rotateleft.png</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="spinDivCount">
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>560</width>
           <height>0</height>
          </size>
         </property>
         <property name="title">
          <string>3D colormap</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="topMargin">
           <number>5</number>
          </property>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>User Define</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboCm5">
            <item>
             <property name="text">
              <string>hsv</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>jet</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Perceptually Uniform</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboCm1"/>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Sequential colormap</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboCm2"/>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Sequential(2) colormap</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboCm3"/>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Diverging colormap</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboCm4"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>560</width>
           <height>0</height>
          </size>
         </property>
         <property name="title">
          <string>3D subplot</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="verticalSpacing">
           <number>10</number>
          </property>
          <item row="1" column="1">
           <widget class="QCheckBox" name="chkBox3D_gridOn">
            <property name="text">
             <string>Grid</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="chkBox3D_invertZ">
            <property name="text">
             <string>Flip Z</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="chkBox3D_axisOn">
            <property name="text">
             <string>Axies</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QComboBox" name="combo3D_type">
            <item>
             <property name="text">
              <string>surface</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>wireframe</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>scatter</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>560</width>
           <height>0</height>
          </size>
         </property>
         <property name="title">
          <string>2D subplot</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="0">
           <widget class="QComboBox" name="combo2D_color">
            <item>
             <property name="text">
              <string>glasbey</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>fire</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Colormap</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QmyFigureCanvas" name="widgetPlot" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>1200</width>
        <height>1200</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="QtApp/res.qrc">
     <normaloff>:/icons/images/exit_24.ico</normaloff>:/icons/images/exit_24.ico</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="actTightLayout">
   <property name="icon">
    <iconset resource="QtApp/res.qrc">
     <normaloff>:/icons/images/39.ico</normaloff>:/icons/images/39.ico</iconset>
   </property>
   <property name="text">
    <string>紧凑布局</string>
   </property>
   <property name="toolTip">
    <string>重新紧凑布局</string>
   </property>
  </action>
  <action name="actSetCursor">
   <property name="icon">
    <iconset resource="QtApp/res.qrc">
     <normaloff>:/icons/images/range.ico</normaloff>:/icons/images/range.ico</iconset>
   </property>
   <property name="text">
    <string>十字光标</string>
   </property>
   <property name="toolTip">
    <string>设置为十字光标</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QmyFigureCanvas</class>
   <extends>QWidget</extends>
   <header location="global">myFigureCanvas</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="QtApp/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
